tools/extras/install_nkf.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Copyright 2017  Johns Hopkins University (author: Daniel Povey)
+# Apache 2.0
```

Comment:
copyright notice should have
```
+# Copyright 2017            Hang Lyu
```
if you wrote it.  And please mention what this tool is for, in a comment near the top.